plugins/by-name/nvim-jdtls/default.nix

```diff
@@ -125,9 +119,23 @@ in
       } // cfg.extraOptions;
     in
     mkIf cfg.enable {
+      assertions = [
+        {
+          assertion = cfg.cmd != null || cfg.data != null;
+          message = "You have to either set the `plugins.nvim-jdtls.data` or the `plugins.nvim-jdtls.cmd` option.";
+        }
+        {
+          assertion = cfg.cmd == null -> cfg.jdtLanguageServerPackage != null;
+          message = ''
+            Nixvim (plugins.nvim-jdtls) You haven't defined a `cmd` or `jdtLanguageServerPackage`.
+
+            The default `cmd` requires `plugins.nvim-jdtls.jdtLanguageServerPackage` to be set.
+          '';
+        }
+      ];
```

Comment:
Same

Reply:
This isn't a breaking change, we default it with a package still. We're just creating an assertion for people that disable that package.

Reply:
Unless I misunderstood lib.mkPackageOption, that is. I thought it defaulted to the pkg name you specify.

Reply:
My mistake. I spotted the assertion but didn't look close enough at what it was doing!

Reply:
All good, I was real confused for a minute :P